ui/apps/platform/src/Containers/Violations/ViolationsTablePage.tsx

```diff
@@ -28,6 +29,8 @@ import useURLSearch from 'hooks/useURLSearch';
 import useURLPagination from 'hooks/useURLPagination';
 import useInterval from 'hooks/useInterval';
 import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import FilteredWorkflowSelector from 'Components/FilteredWorkflowSelector/FilteredWorkflowSelector';
+import useFilteredWorkflowState from 'Components/FilteredWorkflowSelector/useFilteredWorkflowState';
```

Comment:
Super, this is example of what the team discussed recently, that cause and effect relationship is clearest if URL hooks are at same level.

One question about naming where my memory fails even after reviewing similar recent work: do hooks that affect URL also have URL in their names?

Reply:
Yea, we can make it easier to discern that the hook affects the URL state by having the URL in the name. That's a good idea. If we don't do that already, I think its a good pattern.